tools/integration_tests/operations/operations_test.go

```diff
@@ -178,6 +178,12 @@ func TestMain(m *testing.M) {
 	mountConfigFlags := createMountConfigsAndEquivalentFlags()
 	flagsSet = append(flagsSet, mountConfigFlags...)
 
+	// gRPC tests will not run in TPC environment
+	if setup.TestOnTPCEndPoint() && testing.Short() {
```

Comment:
Later, this will cause issue when grpc test will be executed for testing.Short().

I think, we should keep TPC execution independent of Short(). That means, we need to guard in grpc flag in operation package with setup.TestonTPCEndpoint() check.

Reply:
Is this what you were asking for?
I added gRPC flags in case of TPC flag false only.